tst/EnergyPlus/unit/HVACStandaloneERV.unit.cc

```diff
@@ -0,0 +1,95 @@
+// EnergyPlus::Standalone ERV Unit Tests
+
+// Google Test Headers
+#include <gtest/gtest.h>
+
+// EnergyPlus Headers
+#include <EnergyPLus/HVACStandAloneERV.hh>
+#include <EnergyPlus/DataHeatBalance.hh>
+#include <EnergyPlus/DataZoneEquipment.hh>
+#include <EnergyPlus/DataSizing.hh>
+#include <EnergyPlus/UtilityRoutines.hh>
+#include <ObjexxFCL/gio.hh>
+#include <EnergyPlus/SQLiteProcedures.hh>
+
+
+using namespace EnergyPlus;
+using namespace EnergyPlus::HVACStandAloneERV;
+using namespace ObjexxFCL;
+using namespace EnergyPlus::DataHeatBalance;
+using namespace DataGlobals;
+using namespace EnergyPlus::DataZoneEquipment;
+using namespace EnergyPlus::DataSizing;
+
+
+TEST( SizeStandAloneERVTest, Test1 )
+{
+
+	int write_stat;
+	// Open the Initialization Output File (lifted from SimulationManager.cc)
+	OutputFileInits = GetNewUnitNumber( );
+	{ IOFlags flags; flags.ACTION( "write" ); flags.STATUS( "UNKNOWN" ); gio::open( OutputFileInits, "eplusout.eio", flags ); write_stat = flags.ios( ); }
+//	eso_stream = gio::out_stream( OutputFileStandard );
+
+	//CreateSQLiteDatabase(); (lifted from SimulationManager.cc)
+	try {
+		EnergyPlus::sqlite = std::unique_ptr<SQLite>( new SQLite( ) );
```

Comment:
@mbadams5 The routine being tested, SizeStandaloneERV, calls ReportSizingOutput which checks to add an entry to the sqlite output:  
`if ( sqlite->writeOutputToSQLite() ) sqlite->addSQLiteComponentSizingRecord( CompType, CompName, VarDesc, VarValue );
That line failed with a file error before I added this code to set up the sqlite database during the unit test.  Does this mean that every run is setting up the database even if it's never used?  Is there a way to initialize sqlite->writeOutputToSQLite() to false so unit tests don't trip on this.